src/main/java/uk/gov/hmcts/divorce/common/notification/SoleAppliedForFinalOrderNotification.java

```diff
@@ -0,0 +1,99 @@
+    public static final String WILL_BE_CHECKED_WITHIN_14_DAYS = "will be checked within 14 days";
+    public static final String NOW_PLUS_14_DAYS = "now plus 14 days";
+
+    @Autowired
+    private CommonContent commonContent;
+
+    @Autowired
+    private Clock clock;
+
+    @Autowired
+    private NotificationService notificationService;
+
+    @Autowired
+    private HttpServletRequest request;
+
+    @Autowired
+    private CcdAccessService ccdAccessService;
+
+    @Override
+    public void sendToApplicant1(final CaseData caseData, final Long id) {
+
+        if (ccdAccessService.isApplicant1(request.getHeader(AUTHORIZATION), id)) {
+            log.info("Sending Applicant 1 notification informing them that they have applied for final order: {}", id);
+            notificationService.sendEmail(
+                caseData.getApplicant1().getEmail(),
+                SOLE_APPLIED_FOR_FINAL_ORDER,
+                applicant1TemplateVars(caseData, id),
+                caseData.getApplicant1().getLanguagePreference()
+            );
+        }
+    }
+
+    @Override
+    public void sendToApplicant2(final CaseData caseData, final Long id) {
+
+        if (!ccdAccessService.isApplicant1(request.getHeader(AUTHORIZATION), id)) {
+            log.info("Sending Applicant 2 notification informing them that they have applied for final order: {}", id);
+            notificationService.sendEmail(
+                caseData.getApplicant2EmailAddress(),
+                SOLE_APPLIED_FOR_FINAL_ORDER,
+                applicant2TemplateVars(caseData, id),
+                caseData.getApplicant2().getLanguagePreference()
+            );
+        }
+    }
+
+    private Map<String, String> applicant1TemplateVars(CaseData caseData, Long id) {
+        Map<String, String> templateVars =
+            commonContent.mainTemplateVars(caseData, id, caseData.getApplicant1(), caseData.getApplicant2());
+
+        boolean isFinalOrderEligible = caseData.getFinalOrder().getDateFinalOrderNoLongerEligible().isAfter(LocalDate.now(clock));
+
+        templateVars.put(WILL_BE_CHECKED_WITHIN_2_DAYS, isFinalOrderEligible ? CommonContent.YES : CommonContent.NO);
+        templateVars.put(WILL_BE_CHECKED_WITHIN_14_DAYS, !isFinalOrderEligible ? CommonContent.YES : CommonContent.NO);
+        templateVars.put(NOW_PLUS_14_DAYS, !isFinalOrderEligible ? getNowPlus14Days() : "");
+
+        return templateVars;
+    }
+
+    private Map<String, String> applicant2TemplateVars(CaseData caseData, Long id) {
+        Map<String, String> templateVars =
+            commonContent.mainTemplateVars(caseData, id, caseData.getApplicant2(), caseData.getApplicant1());
+
+        templateVars.put(WILL_BE_CHECKED_WITHIN_2_DAYS, CommonContent.NO);
+        templateVars.put(WILL_BE_CHECKED_WITHIN_14_DAYS, CommonContent.YES);
+        templateVars.put(NOW_PLUS_14_DAYS, getNowPlus14Days());
+
+        return templateVars;
+    }
+
+    private String getNowPlus14Days() {
```

Comment:
is this not just the dueDate?

Reply:
Have talked with Sanchita and decided that we'll keep the due date separate.